build

```diff
@@ -4,7 +4,11 @@
 
 ORG_PATH="github.com/square"
 REPO_PATH="${ORG_PATH}/certstrap"
-BUILD_TAG="${TRAVIS_TAG:-$(git describe --all | sed -e's/.*\///g')}"
+BUILD_TAG="${TRAVIS_TAG:-1.99.99-dev.$(git rev-parse HEAD | head -c 8)}"
```

Comment:
Maybe just do dev-$SHA instead of 1.99.99?

Reply:
I was trying to keep it semver-like, but it probably doesn't matter.

Reply:
I feel like it's better to avoid looking like semver if it's not actually meaningful.  I do not have a a strong opinion here.